src/helpers.js

```diff
@@ -206,19 +206,26 @@ function cloner(deep, item) {
   if (type.object(item)) {
     const o = {};
 
-    let k;
+    let i, l, k;
 
     // NOTE: could be possible to erase computed properties through `null`.
-    for (k in item) {
+    const props = Object.getOwnPropertyNames(item);
+    for (i = 0, l = props.length; i < l; i++) {
+      k = props[i];
       if (type.lazyGetter(item, k)) {
         Object.defineProperty(o, k, {
           get: Object.getOwnPropertyDescriptor(item, k).get,
-          enumerable: true,
+          enumerable: Object.getOwnPropertyDescriptor(item, k).enumerable,
```

Comment:
Aren't we in the case of the lazyGetter here (namely, the monkeys)? So do we need to check the value of this item's enumerable property?